js/components/interface/flexLayout2/src/view/BorderButton.tsx

```diff
@@ -79,7 +80,13 @@ export class BorderButton extends React.Component<IBorderButtonProps, any> {
         let leadingContent = undefined;
 
         if (node.getIcon() !== undefined) {
-            leadingContent = <img src={node.getIcon()}/>;
+            if (node.getIcon().startsWith('fa ')) {
```

Comment:
Do we really need to alterate flexlayout for this since we can already achieve this with onRenderTabSet and inject whatever icon we want?

Reply:
@rodriguez-facundo

Reply:
Removed